### PR TITLE
updat envoy and remove wavm

### PR DESCRIPTION
**Pull request by zirain**

### CODE REVIEW

File: bazel/extension_config/extensions_build_config.bzl [hunk 1]
```diff
@@ -507,7 +506,7 @@ ISTIO_ENABLED_CONTRIB_EXTENSIONS = [
     "envoy.filters.network.sip_proxy",
     "envoy.filters.sip.router",
     "envoy.tls.key_providers.cryptomb",
-    "envoy.tls.key_providers.qat",
+    # "envoy.tls.key_providers.qat", # TODO enable this later
```
Comment (zirain) [inline bazel/extension_config/extensions_build_config.bzl:509]: @rveerama1 PTAL

Comment (zirain) [reply bazel/extension_config/extensions_build_config.bzl:509]: CI is broken, please send a PR to enable qat later.

File: extensions/stackdriver/log/logger_test.cc [hunk 2]
```diff
@@ -369,9 +369,9 @@ TEST(LoggerTest, TestWriteLogEntryRotation) {
   flatbuffers::FlatBufferBuilder local, peer;
   std::unordered_map<std::string, std::string> extra_labels;
   auto logger = std::make_unique<Logger>(nodeInfo(local), std::move(exporter), extra_labels, 1200);
-
+  const ::Wasm::Common::FlatNode& peer_node_info = peerNodeInfo(peer);
```
Comment (zirain) [inline extensions/stackdriver/log/logger_test.cc:372]: cc @lei-tang

**Review by lei-tang — APPROVED** [review]

File: bazel/extension_config/extensions_build_config.bzl (hunk 1 above)
Comment (zirain) [reply bazel/extension_config/extensions_build_config.bzl:510]: @soulxu PTAL

Comment (soulxu) [reply bazel/extension_config/extensions_build_config.bzl:510]: @zirain thanks! let me check what happened

**Pull request merged**